libr/bin/p/bin_elf.inc

```diff
@@ -1042,9 +1050,9 @@ static void lookup_symbols(RBinFile *bf, RBinInfo *ret) {
 				ret->lang = "rust";
 			}
 		}
-		symbols_list->free = r_bin_symbol_free;
-		r_list_free (symbols_list);
 	}
+	symbols_list->free = r_bin_symbol_free;
```

Comment:
I think this was in the if because otherwise you could access NULL

Reply:
:+1: 

> I think this was in the if because otherwise you could access NULL

:+1: